pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -2907,7 +2908,7 @@ def visit_module(self, node: nodes.Module) -> None:
         self._class_matchers.reverse()
 
     def visit_classdef(self, node: nodes.ClassDef) -> None:
-        """Called when a ClassDef node is visited."""
+        """Apply relevant class matchers on each nodes.ClassDef."""
```

Comment:
Typo here.

Reply:
Correction - see above, probably not a typo.  But maybe 'Apply relevant matchers to a classdef node.' would read better.